applications/wizard.py

```diff
@@ -36,6 +36,17 @@ def get_next_page_key(self, key):
             if page.status == UNSTARTED:
                 return page.key
 
+    def progress_pc(self):
+        """Return user's progress through wizard as a percentage."""
+
+        num_pages = num_pages_done = 0
+        for page in self.get_pages():
+            num_pages += 1
+            if page.status != UNSTARTED:
+                num_pages_done += 1
```

Comment:
This took me a couple of read throughs to get my head around, I think it's because it's building the two counts at the same time.  Do you think this is more readable?

```
num_pages = len([p for p in self.get_pages()])
num_pages_done = len([p for p in self.get_pages() if p.status != UNSTARTED])
```

I'm not sure I like there being two loops, but it's unlikely to be a performance issue at this scale!